<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_drug_form</class>
 <widget class="QWidget" name="new_drug_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Наименование лс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="drug_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Производитель</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="manufacter_combo"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Серийный номер</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="serial_number"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Срок годности</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Цена</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="price"/>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QPushButton" name="ok_btn">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>1</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancel_btn">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>1</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="best_before">
     <property name="inputMask">
      <string>00.00.0000; </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>new_drug_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
